### PR TITLE
Routes for reward and task

### PR DESCRIPTION
**Pull request by cerafinn**
Completed routes for reward and task routes

### CODE REVIEW

File: controllers/api/reward-routes.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    res.json(data)
+  })
+  .catch(err => {
+    res.status(500).json(err)
+  })
+});
+
+// PUT one: /api/rewards/1
+router.put('/:id', (req, res) => {
+  Reward.update({ where: { id: req.params.id } },
```
Comment (BrianCKWang) [inline controllers/api/reward-routes.js]: `Reward.update(`

File: controllers/api/reward-routes.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  {
+    name: req.body.name,
+    cost: req.body.cost,
+    status: req.body.status,
+    created_by_user_id: req.body.created_by_user_id
+  })
```
Comment (BrianCKWang) [inline controllers/api/reward-routes.js]: `}, { where: { id: req.params.id } } )`

File: controllers/api/task-routes.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    res.json(data)
+  })
+  .catch(err => {
+    res.status(500).json(err)
+  })
+});
+
+// PUT one: /api/tasks/1
+router.put('/:id', (req, res) => {
+  Task.update({ where: { id: req.params.id } },
```
Comment (BrianCKWang) [inline controllers/api/task-routes.js]: `Task.update(`

File: controllers/api/task-routes.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  {
+    name: req.body.name,
+    value: req.body.value,
+    status: req.body.status,
+    created_by_user_id: req.body.created_by_user_id
+  })
```
Comment (BrianCKWang) [inline controllers/api/task-routes.js]: `}, { where: { id: req.params.id } })`

**Review by BrianCKWang — CHANGES_REQUESTED** [review]
Made some comments to change for the PUT that fixes the server error

**Comment by cerafinn** [comment]
Reworked the routes and will resubmit new pull request

**Pull request closed without merging**